source/quotes.html.erb

```diff
@@ -22,7 +22,7 @@ description: Supporting quotes from OpenShift Commons participants.
           </li>
           <li>/</li>
           <li>
-            <a href="/quotes.html">Participant Quotes</a>
+            <a href="/quotes.html">Testimonials</a>
```

Comment:
If we're renaming to testimonials, can we change `quotes.html` to `testimonials.html` as well? I prefer the naming to match. We can then include an .htaccess redirect.

Reply:
done

Reply:
I appreciate you updating the filename....even though we have .htaccess in place for the redirect, would it also make sense to rename the breadcrumb here?